taui/src/components/neighborhood-list-info.js

```diff
@@ -9,25 +9,31 @@ export default function NeighborhoodListInfo ({neighborhood}) {
     return null
   }
 
+  const props = neighborhood.properties
+  const crimeQuintile = props['violentcrime_quintile']
+  const edPercentile = props['education_percentile']
+  const houses = props['house_number_symbol']
+  const isSchoolChoice = !!props['school_choice']
+
   return (
     <table className='neighborhood-facts'>
       <tbody>
-        <tr>
+        {!isSchoolChoice && <tr>
           <td className='neighborhood-facts__cell'>{message('NeighborhoodInfo.EducationCategory')}</td>
           <td className='neighborhood-facts__cell'>
-            <Meter value={86} average={58} tier='high' tooltip='hello' />
+            <Meter value={edPercentile} />
           </td>
-        </tr>
+        </tr>}
         <tr>
           <td className='neighborhood-facts__cell'>{message('NeighborhoodInfo.ViolentCrime')}</td>
           <td className='neighborhood-facts__cell'>
-            <Meter value={72} average={66} tier='med' />
+            <Meter value={crimeQuintile} average={3} max={5} />
```

Comment:
I think this needs to get inverted now that we're calling it "Public Safety" rather than "Crime". If I prioritize safety above all else in the weighting, the first page of results shows zip codes with public safety in the lowest quintile, and shaded red, which implies they're relatively unsafe, but I think it's actually showing their crime rate quintile, meaning they're relatively safe. I Googled a few of the zip codes and it does seem like the ones with the higher value here are relatively less safe than the ones with lower values.

So I think we need to subtract from 5 or find some other way to flip the meaning of the value before displaying it so that this graph (for example) shows up with two big green bars:
<img width="390" alt="Screen Shot 2019-06-06 at 1 06 24 PM" src="https://user-images.githubusercontent.com/447977/59052120-5c64f600-885c-11e9-90a6-4b86f2f5ad95.png">

Reply:
From the spreadsheet header for the column: "used to inform public safety symbol (lower is better). 2 and above green, 3 lighter green, 4 yellow, 5 orange."

Reply:
The actual range in the source data appears to be 0-5, not 1-5 or 0-4.